src/store/reducers/network.js

```diff
@@ -169,58 +177,45 @@ const syncNetworkEpic = () => async (dispatch, getStore) => {
  *
  */
 
-const CheckNetwork = createPromiseActions(CHECK_NETWORK);
+const CheckNetworkAction = createPromiseActions(CHECK_NETWORK);
 
-/**
- * @dev Here we create 3 actions for checking the network status
- * @type {{pending, fulfilled, rejected}|*}
- */
-const subscribeLatestBlockFilter = createPromiseActions(
-  'NETWORK/SUBSCRIBE_LATEST_BLOCK_FILTER',
-);
 
 /**
  * @dev We get latest mined block number
  */
 const getLatestBlockNumber = createAction(
   'NETWORK_GET_LATEST_BLOCK_NUMBER',
-  async () => new Promise((resolve, reject) =>
-    web3.eth.getBlockNumber((e, latestBlockNumber) => {
-      if (e) {
-        reject(e);
-      }
-      else {
-        resolve(latestBlockNumber);
-      }
-    }),
-  ),
+  async () => window.web3p.eth.getBlockNumber(),
 );
 
 const getLatestBlock = createAction(
   'NETWORK_GET_LATEST_BLOCK',
-  async () => new Promise((resolve, reject) =>
-    web3.eth.getBlock('latest', (e, res) => {
-      if (e) {
-        reject(e);
-      }
-      else {
-        resolve(res);
-      }
-    }),
-  ),
+  async () => window.web3p.eth.getBlock('latest'),
+);
+
+
+/**
+ * @dev Here we create 3 actions for checking the network status
+ * @type {{pending, fulfilled, rejected}|*}
+ */
+const subscribeLatestBlockFilterAction = createPromiseActions(
```

Comment:
Probably would be better not to put `Action` as suffix.

Reply:
I specifically added this so it would be clear that this is an action

Reply:
Do we care if it's an action or not ? I mean  all you care is to subsscribe to this. What it does is in the implementation details

Reply:
Ok. ill remove this suffix then.

Reply:
done